Makefile

```diff
@@ -130,8 +130,21 @@ image.buildx.verify:
 		docker buildx version; \
 	fi
 
+BUILDX_CONTEXT = gateway-api-builder
+BUILDX_PLATFORMS = linux/amd64,linux/arm64
+
+# Setup multi-arch enviroment
+# Recreate a buildx context for multi-arch images.
+.PHONY: image.multiarch.setup
+image.multiarch.setup: image.buildx.verify
+	@if [ "$(shell uname)" == "Linux" ]; then \
+		docker run --rm --privileged multiarch/qemu-user-static --reset -p yes; \
+	fi
+	docker buildx rm $(BUILDX_CONTEXT) || :
+	docker buildx create --use --name $(BUILDX_CONTEXT) --platform "${BUILDX_PLATFORMS}"
```

Comment:
Comments here explaining what these are doing could go a long way for future code readers as well.

Reply:
Done.